synergy-design-system/synergy-design-system
# fix: 🤔 Build: Use prettier for formatting generated framework wrappers

# Pull Request

## 📖 Description

This PR adjusts our builds to use prettier instead of eslint for prettifying the created wrapper code. This fastens up the build step significantly and removes some now unneeded dependencies for eslints vue configuration.

As can be seen at the following screenshot, times for builds have lowered from `32sec` to `6sec`, making it ~ 5 faster than before.

<img width="824" alt="Bildschirmfoto 2024-09-16 um 08 36 24" src="https://github.com/user-attachments/assets/3fc27a8e-9383-4d38-a406-fabff2852662">

### 🎫 Issues

Closes #593 

## 👩‍💻 Reviewer Notes

This has to be a fix, because it adjusts the output of the generated wrappers. Functionality wise, it may have been a `chore`, but as the code itself has changed, I think we should opt for a fix here.

## 📑 Test Plan

- Check out this branch (absolutely needed this time!)
- Start via a clean state via `pnpm clean`
- See how fast the build is now :)

## ✅ DoD
